### DISCUSSION TITLE
Discord Oynuyor Durumunuzu GitHub Profilinizde Gösterin!

### DISCUSSION THREAD

**Discussion by barbarbar338** [barbarbar338/webpage#19] [Posts]
# Discord Oynuyor Durumunuzu GitHub Profilinizde Gösterin!
Bildiğiniz üzere [GitHub](https://github.com) üyelerini profillerini güzelleştirmeleri, kendileri hakkında bilgi vermeleri ve bununla havalı şeyler yapmaları için üyelerin profillerine bir markdown dosyası koymalarına izin veriyor, işte benimki şu şekilde:

![Profil Readme Dosyası](https://storage.338.rocks/blog/discord-oynuyor-durumunuzu-github-profilinizde-gosterin/307976243834388484.png)

Yukarıda da gördüğünüz gibi bu dosya bir markdown dosyası olduğu için istediğimiz gibi görseller, linkler, listeler vb. elementler ekleyebiliyoruz. Discord oynuyor durumunu gösterme olayı işte tam da bu sayede oluyor. Benim profilimde biraz aşağıya inerseniz şu şekilde bir kart görebilirsiniz (hatta bu kartı buraya da ekleyebiliyorum):

[![Discord Presence](https://lanyard-profile-readme.vercel.app/api/952574663916154960)](https://discord.com/users/952574663916154960)

Çok havalı değil mi? Discord'da anlık olarak yaptıklarınız GitHub profilinizde veya daha havalısı KENDİ SİTENİZDE (bunun için farklı bir post hazırlamayı düşünüyorum, takipte kalın) gösterebiliyorsunuz! Şimdi bunu nasıl yapacağınızdan bahsedelim.

# Kurulum
İlk olarak kullanacağımız kartı hazırlayan [Conrad](https://github.com/cnrad)'a teşekkür ediyorum. Projenin GitHub linkine [buradan](https://github.com/cnrad/lanyard-profile-readme) ulaşabilirsiniz. Bu kullanacağımız kart [Phineas](https://github.com/Phineas)'ın [lanyard](https://github.com/Phineas/lanyard) adlı projesine dayanarak hazırlanmış bir sistem, [Phineas](https://github.com/Phineas)'a da teşekkürlerimizi iletelim! 

Bu sistemden yararlanabilmek için ilk olarak [lanyard](https://github.com/Phineas/lanyard)'ın Discord sunucusuna katılmanız gerekiyor böylece [lanyard](https://github.com/Phineas/lanyard)'ın botu sizin oynuyor durumunuzu takip edip bunu bir rest API'ye gönderebilecek. [lanyard](https://github.com/Phineas/lanyard)'ın sunucusuna [buradan](https://discord.gg/lanyard) ulaşabilirsiniz. Ve neredeyse her şey tamam! Şimdi ise kontrol etmek için tarayıcıdan `https://lanyard.rest/v1/users/DISCORD_IDNIZ` URL'sine gidip profiliniz takip ediliyor mu diye bakabilirsiniz. Eğer karşınıza aşağıdaki gibi bir çıktı geliyorsa ön hazırlıklar tamam demektir:

![Çıktı](https://storage.338.rocks/blog/discord-oynuyor-durumunuzu-github-profilinizde-gosterin/307978972086534149.png)

Şimdi ise kartımızı profilimize eklemeye geldi sıra. Aşağıdaki kodu profilinize ekledikten sonra her şey tamamdır:

```
[![Discord Presence](https://lanyard-profile-readme.vercel.app/api/DISCORD_ID)](https://discord.com/users/DISCORD_ID)
```

Bu kartı profil readme dosyanıza eklediğinizde aşağıdaki gibi bir kart çıkacak:

[![Discord Presence](https://lanyard-profile-readme.vercel.app/api/952574663916154960)](https://discord.com/users/952574663916154960)

Ve bu kadar, artık Discord durumunuz anlık olarak GitHub profilinizde sergileniyor!

Yukarıdaki kart en temel ve basit kart örneği. [Buradan](https://github.com/cnrad/lanyard-profile-readme#options) karta ekleyebileceğiniz diğer ayarları da görebilirsiniz, örneğin arkaplan değiştirmek gibi.

**Comment by bilalaslam1122** [comment]
sfd
